arches_component_lab/src/arches_component_lab/widgets/FileListWidget/components/FileListWidgetViewer.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { Image, Galleria } from "primevue";
+import type { FileReference } from "@/arches_component_lab/widgets/types.ts";
+
+const props = defineProps<{
+    value: FileReference[] | null | undefined;
+    widgetData: {
+        config: object;
+    };
+}>();
+
+const images = computed(() => {
+    const val = props.value?.map((x) => {
+        return {
+            thumbnailImageSrc: `${x.url}`,
+            itemImageSrc: `${x.url}`,
+            alt: x.altText,
+            title: x.title,
+        };
+    });
+    console.log(val);
+    return val;
+});
+</script>
+
+<template>
+    <Galleria
+        :value="images"
+        :show-thumbnails="images && images?.length > 1"
```

Comment:
nit: I'd make a computer out of this, to keep logic out of the template. that way it can be used below as well.